bin/protoc

```diff
@@ -16,7 +16,12 @@ def platform_arch
 end
 
 def exec_protoc_for_os(os)
-  exec("#{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[platform_arch]}.exe", *ARGV)
+  case Platform::OS
+    when :win32
+      exec("#{File.dirname(__FILE__)}/protoc-#{Protoc::PROTOBUF_VERSION}-#{os}-#{PLATFORM_TO_PROTOC_ARCH[platform_arch]}.exe", *ARGV)
```

Comment:
Is it better to define the filename once, and just append .exe if windows? That seems less error prone.

Reply:
Pretty sure avoiding this `case` was the reason the binaries all had '.exe' on the end in the first place.